src/new-client/src/Web/TearOutSection/TearOutComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import { HeaderAction } from "@/components/styled"
+import { PopOutIcon } from "@/components/icons/PopOutIcon"
+import { tearOutSection } from "@/Web/TearOutSection/state"
+import { PopInIcon } from "@/components/icons/PopInIcon"
+import {
+  TornOutTileContext,
+  TornOutTradeContext,
+  TornOutAnalyticsContext,
+} from "@/Web/context"
+
+export const TearOutComponent: React.FC<{ section: string }> = ({
+  section,
+}) => {
+  var context = TornOutTileContext
```

Comment:
Minor - Avoid the usage of `var`.
`const` for everything unless you're going to reassign the value - `let`

Reply:
Done.